x-pack/plugins/infra/server/lib/alerting/inventory_metric_threshold/inventory_metric_threshold_executor.ts

```diff
@@ -83,6 +83,7 @@ export const createInventoryMetricThresholdExecutor = (libs: InfraBackendLibs) =
       spaceId,
       startedAt,
       rule: { id: ruleId, tags: ruleTags },
+      getTimeRange,
```

Comment:
@maryam-saeidi , the delay value is coming from `getTimeRange` function, and this function will handle the serverless delay (read it from the modal)/stateful immediate execution. So we only need to read the value. 
Is that a correct assumption?

Reply:
Yes, that logic was implemented [here](https://github.com/elastic/kibana/pull/168735)